glyphs-reader/src/font.rs

```diff
@@ -2011,6 +2011,41 @@ impl TryFrom<RawShape> for Shape {
     }
 }
 
+// Direct port from
+// <https://github.com/fonttools/fonttools/blob/b7509b2/Lib/fontTools/misc/transform.py#L65-L77>
+const EPSILON: f64 = 1e-15;
+const ONE_EPSILON: f64 = 1.0 - EPSILON;
+const MINUS_ONE_EPSILON: f64 = -1.0 + EPSILON;
```

Comment:
since these are only used in side `norm_sin_cos` I would define them in there? It just makes it clear in the future that they're just for that fn, and makes it less likely they get separated in some future patch or refactor.

(I'd personally also define `norm_sin_cos` inside of `normalized_rotation` for the same reason, but that's a more contentious choice)